### PR TITLE
Update for the change in the Laravel Handler

### PR DESCRIPTION
**Pull request by designvoid**

### CODE REVIEW

File: src/JsonHandler.php [hunk 1]
```diff
@@ -11,11 +11,11 @@ trait JsonHandler
      * Handle the json response. Check if exception is treated. If true call
      * the specific handler. If false set the default response to be returned.
      *
-     * @param  \Exception $exception
+     * @param  \Throwable  $exception
      *
      * @return \Illuminate\Http\JsonResponse
      */
-    public function jsonResponse(Exception $exception)
+    public function jsonResponse(Throwable $exception)
```
Comment (sfelix-martins) [inline src/JsonHandler.php:18]: @designvoid
wouldn't it be necessary to import the Throwable class?

Comment (sfelix-martins) [reply src/JsonHandler.php:18]: Please, can you remove the unused Exception import?

**Comment by sfelix-martins** [comment]
@designvoid thanks for your contribution, please check the review.

**Comment by designvoid** [comment]
@sfelix-martins I can't see any review?

**Review by sfelix-martins — COMMENTED** [review]
> @sfelix-martins I can't see any review?

Please, check [here](https://github.com/sfelix-martins/json-exception-handler/pull/9/files/d0b98edb2380489ddd36d62cb43336db48855a71)

File: src/JsonHandler.php (hunk 1 above)
Comment (designvoid) [reply src/JsonHandler.php:18]: Sure thing - that should be fixed now - literally no idea how I missed that

**Comment by designvoid** [comment]
Is this all ok now @sfelix-martins ?

**Comment by sfelix-martins** [comment]
> Is this all ok now @sfelix-martins ?

@designvoid I will accept it ASAP. I'm very busy lately...

Probably today I'll publish a release with your contribution.

**Comment by relocity-rd** [comment]
@sfelix-martins Any update on merging this PR that @designvoid contributed to?

**Pull request merged**